modules/dnn/tutorials/tutorial_dnn_halide_scheduling.markdown

```diff
@@ -75,9 +77,8 @@ fc8:
 
 ## Automatic scheduling
 Based on manual scheduling experience, proposed way to schedule layers
-automatically. Just skip scheduling file path argument at ```cv::dnn::Net::compileHalide```
-for let DNN schedule your network. Sometimes it might be even better
-than manual scheduling.
+automatically. Just skip call of ```cv::dnn::Net::setHalideScheduler``` for let
```

Comment:
"Based on manual scheduling experience, proposed way to schedule layers automatically." - it seems that something's missed in this sentence